app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingItem.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.keniphhomemaintenance.dwellings
+
+import androidx.compose.foundation.layout.Column
+// import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.example.keniphhomemaintenance.ui.Screen
+
+val dwellingsViewModel = DwellingsViewModel()
+
+@Composable
+fun DwellingItem(dwelling: Dwelling, navController: NavController) {
+    Card(
+        modifier = Modifier
+            .padding(horizontal = 8.dp, vertical = 8.dp)
```

Comment:
Can store dimensions in a dimens.xml file

Reply:
Noted, thank you

Reply:
ended up not having dimens.xml file, all dimensions in mine are static constants

Reply:
These are not static constants; these are hard-coded values. static constants are variables that never change value, i.e. const val THING = "thing".

Reply:
They still look like plain numbers, no?

Reply:
Dimens.kt created